core/src/main/java/io/kestra/core/runners/ExecutorService.java

```diff
@@ -104,7 +109,7 @@ public Executor process(Executor executor) {
             executor = this.handleKilling(executor);
 
             // process next task if not killing or killed
-            if (executor.getExecution().getState().getCurrent() != State.Type.KILLING && executor.getExecution().getState().getCurrent() != State.Type.KILLED) {
+            if (executor.getExecution().getState().getCurrent() != State.Type.KILLING && executor.getExecution().getState().getCurrent() != State.Type.KILLED && executor.getExecution().getState().getCurrent() != State.Type.QUEUED) {
```

Comment:
This should not happened as the execution has been sent to queueing

Reply:
It happen because I send the `QUEUED` on the execution queue (that will index the document)